labs/sysdev-u-boot-bbb/sysdev-u-boot-bbb.tex

```diff
@@ -377,20 +379,42 @@ \subsection{Network configuration on the PC host}
 
 \section{Setting up the TFTP server}
 
-Let's install a TFTP server on your development workstation:
+Let's install a TFTP (Trivial File Transfer Protocol) server on your development workstation:
 
 \begin{verbatim}
 sudo apt install tftpd-hpa
 \end{verbatim}
 
 You can then test the TFTP connection. First, put a small text file in
-the directory exported through TFTP on your development
+the directory exported through TFTP (\code{/srv/tftp} by default) on your development
```

Comment:
There as well, the idea by not providing it is to make sure participants go back to the slides, re-read the information provided there.